ui/src/components/admin/faqs/FaqGrid.js

```diff
@@ -71,10 +281,68 @@ export default function FaqGrid(props) {
         <Switch
```

Comment:
i just realixed the <Heading> here has `s` and `size`. if s is not size, what is it?

Reply:
haha i dont know, ill remove it